app/views/theblog/admin/_form.html.haml

```diff
@@ -1,7 +1,10 @@
-= simple_form_for [:admin, item] do |f|
+= simple_form_for [:admin, item], html: { autocomplete: :off } do |f|
```

Comment:
you mean for items and accounts? As I see, the difference is not too big now, so we could keep it as I made